libs/core/src/components/pds-loader/readme.md

```
@@ -34,12 +34,14 @@
 ### Used by
 
  - [pds-button](../pds-button)
```

markdownlint-cli2 (0.20.0)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)